.github/ISSUE_TEMPLATE/create-new-doctype-template.md

```diff
@@ -6,22 +6,51 @@ labels: templates
 assignees: ''
 
 ---
+# Title
+{Keep the title short, yet descriptive.
+Here are some examples:
+* Create a new {doc_type} template 
+* Align the existing {doc_type} template to new contributing guidelines
+* Include a quality checklist for the quickstart template }
 
-# Create {doctype} template
+## Before you begin
+{Optional: In this section specify any pre-requisites, background knowledge that the templateer should know before working on a template.}
+
+1. Join The Good Docs [slack workspace](https://thegooddocs.slack.com/). 
+2. Familiarize yourself with basic Git commands and understand the workflow for contributing on GitHub.
+3. Read the [template contributing process](https://github.com/thegooddocsproject/templates/blob/dev/CONTRIBUTING.md#overview-of-the-template-writing-phases).
+4. You are also strongly encouraged to join one of the [working groups](https://thegooddocsproject.dev/working-group/) to get valuable support from the community.
 
 ## Description
+{Provide specific details about the template. You could adopt a user story approach to describe a new {doc_type} template.  
+
+_As a [type of user] I want [my goal] so that [my reason]._  
+
+Example:
+_As a developer, I want an installation template to document the install instructions for my application._ 
+
+Keep the following points in mind while writing the description:
+* If applicable, always include the relevant people using `@{Username}` who can support in resolving the issue. It can be someone who has previously worked on the template. Also, indicate the relevant slack channel where the templateer can reach out to the larger community. 
+* Use bullet points and subheadings to structure complex details instead of big paragraphs.
+* Add links to any document references.
+}
+
 ### Doc Type Name: {doc_type}
+
 ### Directory: https://github.com/thegooddocsproject/templates/{directory_name}
-### Sub-tasks
-1. Create {doc_type}-template.md and write content for it
-2. Create {doc_type}-guide.md and write content for it
 
-{Optional: add any further explanation if the issue is not self-explanatory}
+## Tasks
+{Optional: Specify a broad list of tasks to help the templateer get started with the template.}
+1. Research examples and identify best practices for writing a {doc_type}.
+2. Create {doc_type}-template.md and write content for it.
+3. Create {doc_type}-template-guide.md and write content for it.
```

Comment:
Do we mention the optional deep dive guide? How often will templateers write one, do you think? Is it worth mentioning in this template?

Reply:
I have made it generic instead of specifying each deliverable. Let me know if this is ok.

1. Research examples and identify best practices for writing a {content_type}.
2. Create drafts of the template set deliverables.
3. Contact the template leads to let them know that your draft is ready for community review. 
4. Get feedback on the drafts from the community.
5. After making revisions, submit a pull request.

Reply:
I like your change. Looks good to me!